openbsd-compat/openssl-compat.c

```diff
@@ -66,6 +69,87 @@ ssh_compatible_openssl(long headerver, long libver)
 	return 0;
 }
 
+int
+get_wcwidth(wchar_t wc) {
+	if (0x20 <= wc && wc <= 0x7e)
```

Comment:
can you please add some comments, what this method is doing and how to interpret the hex codes (may be a URL that has these values will help us)..